languages/tr.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Please pull the latest bc we recently merged a PR for Turkish language support

Reply:
Several minutes ago i was updated the PR with this fixes.